File: backer/godotask/model/qualitative_label.go
package model

import (
	"time"
)

// QualitativeLabel - プロセス最適化記録
type QualitativeLabel struct {
  ID int `gorm:"type:varchar(255);primaryKey" json:"id"`
  TaskID    int            `gorm:"index"`
  UserID    int           `gorm:"index"`
  Content   string         `gorm:"type:text"` // ラベル内容
  Category  string         `gorm:"index"`     // ラベルのカテゴリ
  CreatedAt time.Time
  UpdatedAt time.Time
}

File: fronter/src/model/qualitativeLabel.ts

export interface QualitativeLabel {
  id: string;
  task_id: number;
  user_id: number;
  Content: string;
  Category: string;
  created_at: string; // ISO date string
  updated_at: string; // ISO date string
}

export interface AddQualitativeLabel {
  task_id: number;
  user_id: number;
  Content: string;
  Category: string;
}
